src/main.h

```diff
@@ -88,6 +88,9 @@ static const unsigned int BLOCK_STALLING_TIMEOUT = 2;
 /** Number of headers sent in one getheaders result. We rely on the assumption that if a peer sends
  *  less than this number, we reached its tip. Changing this value is a protocol upgrade. */
 static const unsigned int MAX_HEADERS_RESULTS = 2000;
+/** Maximum age (in blocks) of blocks we're willing to serve as compact blocks to peers
+ *  when requested. For older blocks, a regular BLOCK response will be sent. */
+static const int MAX_COMPACT_BLOCK_DEPTH = 5;
```

Comment:
It's been too long since I've looked at this code, but this sure seems close to `MAX_CMPCTBLOCK_DEPTH`